staff/vic/middleware-task-api/src/api/apiTask.js

```diff
@@ -0,0 +1,37 @@
+const axios = require('axios');
+
+const URL = `http://localhost:5000/api`
+
+function __get (path) {
+    return axios.get(path)
+        .then(res => res.data)
+}
+
+function __post (path, text){
+    return axios.post(path, {text})
+}
+
+function __put (path){
+    return axios.put(path)
+}
+
+function __delete (path) {
+    return axios.delete(path)
```

Comment:
MAY you return after .then(res => res.data), as axios encapsulates the response body there